### PR TITLE
Make legacy API and its deprecation more prominent 

### PR DESCRIPTION
**Pull request by davidoliverSP2**
Cherry picks #40

### CODE REVIEW

File: modules/ROOT/pages/actions/legacy-endpoints.adoc [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+====
+The API described in this section of the manual has been deprecated and will be removed in Neo4j 5.0.
+====
+
+For backwards compatibility with neo4j 3.5, it is still possible to use the HTTP API without specifying the database in the URL.
+For example:
+
+`http://localhost:7474/db/data/transaction/commit`.
+
+Since the database is not specified, the statements will be executed against the user's _global default database_.
```
Comment (davidoliverSP2) [inline modules/ROOT/pages/actions/legacy-endpoints.adoc:16]: @gjmwoods Are you happy with this wording for the pre-4.3 versions?

**Comment by stefano-ottolenghi** [comment]
@gjmwoods any updates on this? Can we merge?

**Comment by stefano-ottolenghi** [comment]
Is this still actual? Anything that should be changed here, or can we merge @gjmwoods ?

**Comment by gjmwoods** [comment]
@stefano-ottolenghi Sorry for the _slight_ delay on this one 😅 . Yeah this still looks accurate for pre-4.3 versions and the wording looks correct.

**Pull request merged**